types/resize-observer-browser/resize-observer-browser-tests.ts

```diff
@@ -0,0 +1,23 @@
+function resizeObserverCreates(): void {
+    const resizeObserver: ResizeObserver = new ResizeObserver(entries => {
```

Comment:
nit: could you make the parameter `(entries)` like it was before?

Reply:
Done.

I wish github would show this as a move.  `git show` shows it as a rename + changes.

Reply:
Now it shows the move + diff!  I don't know what I did.